sources/assets/exegol/load_supported_setups.sh

```diff
@@ -175,48 +197,40 @@ function deploy_bloodhound_customqueries_merge() {
 }
 
 function deploy_bloodhound_customqueries_replacement() {
+  colorecho "Replacing BloodHound Exegol Custom Queries"
   # Replace Exegol's customqueries.json file with the merge of ones from my-resources
   local cq_replacement_directory="$my_setup_bh_path/customqueries_replacement"
-
   [[ ! -d "$cq_replacement_directory" ]] && cp -r /.exegol/skel/bloodhound/customqueries_replacement "$cq_replacement_directory"
-
   if [[ -n $(find "$cq_replacement_directory" -type f -name "*.json") ]]; then
       bqm --verbose --ignore-default --output-path "$bqm_output_file" -i "$cq_replacement_directory"
   fi
 }
 
 function deploy_bloodhound() {
+  colorecho "Deplying BloodHound"
```

Comment:
There's a typo

Reply:
Indeed, thank you! Fixed in https://github.com/ThePorgs/Exegol-images/pull/246/commits/4e2d64314788f6e317f944211dff3709a79c8071